packages/messages/src/browser/notification-center-component.tsx

```diff
@@ -67,13 +65,11 @@ export class NotificationCenterComponent extends React.Component<NotificationCen
                         </ul>
                     </div>
                 </div>
-                <PerfectScrollbar className='theia-notification-list-scroll-container'>
-                    <div className='theia-notification-list'>
-                        {this.state.notifications.map(notification =>
-                            <NotificationComponent key={notification.messageId} notification={notification} manager={this.props.manager} />
-                        )}
-                    </div>
-                </PerfectScrollbar>
+                <div className='theia-notification-list'>
```

Comment:
please wrap this in an additional `<div className='theia-notification-list-scroll-container'>` container.